docker/cvmfs-install/Dockerfile.cvmfs-install.centos7

```diff
@@ -29,8 +29,16 @@ RUN yum install -y cvmfs \
 		   mysql \
 		   mysql-devel \
 		   ghostscript \
+           gcc \
```

Comment:
Installing `gcc` *might* be conflicting with the one we use in sft.cern.ch. Any thought on this?
Same with `libstdc++` and `cmake`

Reply:
That should in principle be fine, I suppose, since the ones from the LCG release that is sourced in [`use_gcc82_cvmfs_el7.sh`](https://github.com/iLCSoft/iLCInstall/blob/master/scripts/use_gcc82_cvmfs_el7.sh) take precedence to the ones that are installed system wide.